### ISSUE TITLE
Unable to resolve artifact: Missing: for org.robolectric:robolectric:3.0-SNAPSHOT

### ISSUE THREAD

**Issue by pbibalan** [robolectric/robolectric#1536]
I'm using appcompat-v7 and ActionBarActivity and according to this (https://github.com/robolectric/robolectric/issues/1446)  the issue is resolved in Robolectric 3.0. So I tried adding this
```
androidTestCompile('org.robolectric:robolectric:3.0-SNAPSHOT')
```
to my build.gradle file and run my tests but I get the following dependency issue. 
```
Unable to resolve artifact: Missing:
----------
1) org.robolectric:robolectric-resources:jar:3.0-SNAPSHOT

  Try downloading the file manually from the project website.

  Then, install it using the command: 
      mvn install:install-file -DgroupId=org.robolectric -DartifactId=robolectric-resources -Dversion=3.0-SNAPSHOT -Dpackaging=jar -Dfile=/path/to/file

  Alternatively, if you host your own repository you can deploy the file there: 
      mvn deploy:deploy-file -DgroupId=org.robolectric -DartifactId=robolectric-resources -Dversion=3.0-SNAPSHOT -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  Path to dependency: 
  	1) org.apache.maven:super-pom:pom:2.0
  	2) org.robolectric:shadows-core:jar:18:3.0-SNAPSHOT
  	3) org.robolectric:robolectric-resources:jar:3.0-SNAPSHOT

----------
1 required artifact is missing.

for artifact: 
  org.apache.maven:super-pom:pom:2.0

from the specified remote repositories:
  sonatype (https://oss.sonatype.org/content/groups/public/),
  central (http://repo1.maven.org/maven2)


	at org.apache.maven.artifact.ant.DependenciesTask.doExecuteResolution(DependenciesTask.java:268)
	at org.apache.maven.artifact.ant.DependenciesTask.doExecute(DependenciesTask.java:168)

```
Any suggestions?

**Comment by erd** [comment]
You need to add Sonatype's snapshot repo to your build.gradle:

    repositories {
        maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
    }

**Comment by pbibalan** [comment]
Thanks @erd but I already have that. 
```
buildscript {
    repositories {
        mavenCentral()
        maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
        maven { url 'https://repo.commonsware.com.s3.amazonaws.com' }
        maven { url 'http://download.crashlytics.com/maven' }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:1.0.0'
        classpath 'com.crashlytics.tools.gradle:crashlytics-gradle:1.+'
        classpath 'org.robolectric:robolectric-gradle-plugin:0.14.+'
        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.+'
    }
}

apply plugin: 'android-sdk-manager'
apply plugin: 'com.android.application'
apply plugin: 'crashlytics'
apply plugin: 'robolectric'

repositories {
    mavenCentral()
    maven { url 'http://download.crashlytics.com/maven' }
    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
    maven { url 'https://repo.commonsware.com.s3.amazonaws.com' }
}
```
I'm using the robolectric-gradle-plugin if that changes anything?

**Comment by pbibalan** [comment]
For your reference, just posted on robolectric-gradle-plugin asking about the issue: 
https://github.com/robolectric/robolectric-gradle-plugin/issues/127

**Comment by emilesvt** [comment]
What's the workaround here?  Download and manually install in our maven repo?

**Comment by naoyamakino** [comment]
I am still seeing this issue now. any updates?

**Comment by johannes82** [comment]
Got the same issue. Any updates?

**Comment by ChrisShottonMAS** [comment]
Started seeing this issue also.

**Comment by grub-** [comment]
I also got it. What's the workaround?

**Comment by johannes82** [comment]
My solution is to download the file and install it manually
 $ mvn install:install-file -DgroupId=org.robolectric -DartifactId=shadows-core -Dversion=3.0-20150330.025101-320 -Dclassifier=18 -Dpackaging=jar -Dfile=/path/to/file

**Comment by brsengar** [comment]
I am also getting this issue again...just now..

**Comment by rsavin** [comment]
I've changed emulatedSdk in my Config to Lollipop version code and it worked:
```
@Config(emulateSdk = Build.VERSION_CODES.LOLLIPOP, manifest = Config.NONE)
```

**Comment by erd** [comment]
The SNAPSHOT build is a bit broken at the moment and I won't have a chance to work on it until later tonight. Please try and use the latest RC build - "org.robolectric:robolectric:3.0-rc2" (it may take a few mins for it to sync with Maven Central).

**Comment by WojciechKo** [comment]
Where can I find "org.robolectric:robolectric:3.0-rc2"?

**Comment by snakelizzard** [comment]
Thanks, it works.

**Comment by erd** [comment]
Closing this in favor of #1536 (since they are the same issue).

**Comment by SangWoo-Maeng** [comment]
I've just solve this problem with changing the version number like this.
testCompile "org.robolectric:robolectric:2.4"

The latest version in Github is 2.4. NOT 3.0
see : https://github.com/robolectric/robolectric/releases

**Comment by AlexBonel** [comment]
> (since they are the same issue)

Am I understand correctly that you've closed the issue in favor of the issue itself?

**Comment by nenick** [comment]
I guess he mean https://github.com/robolectric/robolectric/issues/1643 ;)

**Comment by erd** [comment]
@nenick Yep, that's what I meant (I've updated the comment).

**Comment by volodia-chornenkyy** [comment]
I've was able to fix the same issue by overriding Robolectric Maven repository URL. 
Sonatype is not available for some reason and Maven Central moved to HTTPS starting from Jan 15, 2020. So, I've decided to use Maven Central URL but with a secure scheme.

Editing system property which is used by Robolectric RoboSettings class allowed me to use different URL.
`android {
    testOptions {
        unitTests.all {
            systemProperty 'robolectric.dependency.repo.url', 'https://repo1.maven.org/maven2'
        }
    }
}`
